src/ui/components/Library/SidebarFooter.tsx

```diff
@@ -12,17 +12,18 @@ function SidebarFooter({ setModal }: PropsFromRedux) {
   };
 
   return (
-    <div className="flex flex-row space-x-2 bg-gray-700 p-4 cursor-pointer" onClick={handleSettingsClick}>
+    <button
+      className="flex flex-row space-x-2 bg-gray-700 p-4 cursor-pointer"
```

Comment:
don't need `cursor-pointer` we get that for free with the button! :)

Reply:
True, thanks!